torch/onnx/symbolic.py

```diff
@@ -175,7 +175,7 @@ def _try_get_scalar_type(*args):
 # ONNX operator version
 # ---------------------------------------------------------------------
 
-# READ ME BEFORE EDITING _onnx_opset_version:
+# READ ME BEFORE EDITING _default_onnx_opset_version:
 #
```

Comment:
@houseroad - Minor point - it will be nice to have an example of dispatch to different symbolic based on the opset version. Doesn't necessarily have to be in this PR.